ugvc/pipelines/run_no_gt_report.py

```diff
@@ -299,14 +308,143 @@ def run_full_analysis(arg_values):
         eval_table.to_hdf(f"{arg_values.output_prefix}.h5", key=f"eval_{eval_table_name}")
 
 
+
+def run_somatic_analysis(arg_values):
+
+    def _get_signatures_from_sig_log(log_file):
+        line = log_file.readline()
+        while "Composition After Add-Remove" not in line:
+            line = log_file.readline()
+
+        signatures = log_file.readline().strip("\n").split()
+        activity_values = log_file.readline().strip("\n").split()[1:]
+        # parse the signatures names
+        return signatures, activity_values
+
+    # Install reference
+    genInstall.install(arg_values.reference_name)
+
+    # Run SigProfilerAssignment for getting the signatures of the sample
+    vcf_dirname = os.path.dirname(arg_values.input_file)
+    output_dir = arg_values.output_prefix + '/'
+    Analyze.cosmic_fit(samples=vcf_dirname,
+                       output=os.path.join(output_dir,'sbs'),
+                       input_type="vcf",
+                       context_type="96",
+                       genome_build=arg_values.reference_name,
+                       cosmic_version=3.3)
```

Comment:
Maybe to allow compatibility with different versions this parameter can be externalized?